rules/regex-shorthand.js

```diff
@@ -6,7 +6,36 @@ const quoteString = require('./utils/quote-string');
 
 const message = 'Use regex shorthands to improve readability.';
 
+const availableRegexpTreeOptimizations = [
```

Comment:
Can you add a todo comment about simplifying this when https://github.com/DmitrySoshnikov/regexp-tree/pull/200 is released?

Reply:
Why we need that? Why not default to all options and remove disabled items?

Reply:
@fisker As far as I see it, we want to convey what rules we do _not_ want, rather than convey what rules we do want. As the important thing in this case is what isn't included, not what is included. Intention becomes more clear if regexp-tree gets handed such a blacklist directly.

Added a TODO

Reply:
A built in blacklist conveys our intention better + makes any future changes to the set of rules become applied automatically, as it would have been before this PR.